gig/forms.py

```diff
@@ -26,9 +28,29 @@ def __init__(self, **kwargs):
             **kwargs
         )
 
+    def clean_date(self):
+        date = self.cleaned_data['date']
+        if date < timezone.now():
+            raise ValidationError(_('Date must be in the future'), code='invalid date')
+        return date
+
+    def clean_setdate(self):
+        date = self.cleaned_data['date']
+        setdate = self.cleaned_data['setdate']
+        if setdate and setdate < date:
+            raise ValidationError(_('Set date must be later than the start date'), code='invalid end date')
```

Comment:
This may be confusing to users, since the set date can be the same as the start date; it's just the time that has to be later.  (Or actually, not earlier.)  Perhaps, "Set date must not be earlier than start date"?

In the emails the date is referred to as the "call", not the "start".  I don't know if that's a consistent name or not.